R/lasso-development.R

```diff
@@ -176,7 +176,7 @@ LassoDevelopment <- R6Class("LassoDevelopment",
     
     # Fit model and lamda values
     fitGrLasso = NA,
-    fitLogit = NA,
+    modelName = "lasso",
```

Comment:
It will require a little bit of work because the model names also have to be specified in deployment, I'll see what I can do.  Looking ahead, I think there's more functionality of this type we might want to add (see Issue #514)